make/autoconf/jdk-version.m4

```diff
@@ -109,6 +109,33 @@ AC_DEFUN_ONCE([JDKVER_SETUP_JDK_VERSION_NUMBERS],
   fi
   AC_SUBST(MACOSX_BUNDLE_ID_BASE)
 
+  # The launcher name, if any
+  AC_ARG_WITH(launcher-name, [AS_HELP_STRING([--with-launcher-name],
+      [Set launcher name. Part of the -version and -fullversion output. @<:@not specified@:>@])])
+  if test "x$with_launcher_name" = xyes; then
+    AC_MSG_ERROR([--with-launcher-name must have a value])
+  elif [ ! [[ $with_launcher_name =~ ^[[:print:]]*$ ]] ]; then
```

Comment:
It would be helpful to have a comment about the need for the outermost brackets (for m4).

Reply:
Or not, I see the same pattern in the handler for `--with-macosx-bundle-id-base`.

Reply:
I just copied this stuff from vendor-name. What would you like the comment to be?

Reply:
Ah, didn't see your update until now.

Reply:
All the review comments should be addressed then.